Jenkinsfile

```diff
@@ -74,6 +79,10 @@ def runRelease() {
 
     try {
         if ("$EXECUTION_MODE".toString().equals("Publish packages")) {
+            stage('Docker Login') {
```

Comment:
Docker Login --> Docker login

Let's also change the titles of all states to have a capital letter only in the first word.